types/couchbase/index.d.ts

```diff
@@ -1013,6 +1013,20 @@ interface Bucket {
      * @param callback The callback function.
      */
     upsert(key: any | Buffer, value: any, options: UpsertOptions, callback: Bucket.OpCallback): void;
+
+    /**
+     * Stores a document to the bucket.
+     * @param key The target document key.
+     * @param options The options object.
+     */
+    mutateIn(key: string, options?: object): any;
```

Comment:
Use of the `object` keyword requires you to add a `// TypeScript Version: 2.3` comment at the top of the file, per [the readme](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-use-features-from-typescript-21-or-above)